src/Components/WebAssembly/Build/src/Tasks/GenerateBlazorBootJson.cs

```diff
@@ -88,6 +88,9 @@ internal void WriteBootJson(Stream output, string entryAssemblyName)
                         case "assembly":
                             resourceList = resourceData.assembly;
                             break;
+                        case "dynamicAssembly":
```

Comment:
I ended up changing how this works in sdk since a different file is produced once for build and release and it's harder to share code in MSBuild targets. It's nice that we have tests so I should be able to verify these work once we replat